server/block/hash.go

```diff
@@ -136,6 +136,7 @@ const (
 	hashWoodStairs
 	hashWoodTrapdoor
 	hashWool
+	hashSlime
```

Comment:
I would keep this list in alphabetical order - but you should probably be using the blockhash tool for this.

Reply:
Ok I'll change that too